stockflux-search/src/App.js

```diff
@@ -90,10 +90,20 @@ const App = () => {
             <Components.Titlebar />
             <input type="text" className={styles.input} onChange={event => setQuery(event.target.value)} placeholder="Enter stock name or symbol" />
             <div className={styles.containerList}>
-                { !isSearching && results && results.length ?
-                    results.map((result) => (<SearchResult key={result.code} code={result.code} name={result.name} />)) :
-                    <div className={styles.message}>{getMessage(searchState, results)}</div>
-                }
+                <Components.ScrollWrapperY contentChanged={Boolean(results)}>
```

Comment:
Not sure if contentChanged will work as intended here, isn't it just going to run only once after we initially get results?

Reply:
You're right, just tested. Removed the boolean, so every time when we get results the prop would update and trigger overflow recalculation. Fixed -> 08cff915e0c7c162c3c71c72e9adb0e02ba1de6c